docs/pages/_app.js

```diff
@@ -216,7 +216,7 @@ function AppWrapper(props) {
         productIdentifier = {
           metadata: '',
           name: 'Toolpad Core',
-          versions: [{ text: `v0.0.1`, current: true }],
+          versions: [{ text: `v${toolpadPkgJson.version}`, current: true }],
```

Comment:
- Remove the hard coded version and use the same version number as Toolpad Studio, with the assumption that we keep version numbers in tandem.